### ISSUE TITLE
How to reproduce quantitative results in your paper?

### ISSUE THREAD

**Issue by JOANNECYSHEN** [iperov/DeepFaceLab#834]
Thank you very much for the great works!
I would like to reproduce quantitative results in your paper, and I have some questions as below: 

Dataset:
you described "To be statistically significant, we compute the mean and variance of those measurements on the 100 frames (uniform sampling over time) of the first 500 videos in FaceForensics++" 
**Question 1 : As I know, your model support faceswap from src to dst (one face pair at a time). How to feed multi-face pairs into the training model?**

Training setting:
you described "It should be noted that all the videos produced by DeepFaceLab were follow by the same settings with 5.2." 
**Question 2: How to make it "The average training time is restrict within 3 hours" in 5.2? (It might be more clear to me if I know the answer of question 1)**

Result: 
**Question 3:　Table 1 in your paper show the quantitative face swapping results on FaceForensics++. Could I know how to calculate those index: SSIM, perceptual loss, verification, landmarks, pose? Or is it possible to release code to let us reproduce experiments in your paper?**

Looking forward to your reply! Thank you!
Joanne

**Comment by TMBdf** [comment]
You're better off contacting Kunlin Liu (email in the paper) as he is the one that actually wrote it (but I could be wrong) with help of others listed.

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
